content/en/docs/concepts/scheduling-eviction/_index.md

```diff
@@ -1,11 +1,39 @@
 ---
 title: "Scheduling, Preemption and Eviction"
 weight: 90
+content_type: concept
 description: >
   In Kubernetes, scheduling refers to making sure that Pods are matched to Nodes
   so that the kubelet can run them. Preemption is the process of terminating 
   Pods with lower Priority so that Pods with higher Priority can schedule on 
   Nodes. Eviction is the process of proactively terminating one or more Pods on
   resource-starved Nodes.
+no_list: true
 ---
 
+In Kubernetes, scheduling refers to making sure that {{<glossary_tooltip text="Pods" term_id="pod">}}
+are matched to {{<glossary_tooltip text="Nodes" term_id="node">}} so that the
+{{<glossary_tooltip text="kubelet" term_id="kubelet">}} can run them. Preemption
+is the process of terminating Pods with lower {{<glossary_tooltip text="Priority" term_id="pod-priority">}}
+so that Pods with higher Priority can schedule on Nodes. Eviction is the process
+of terminating one or more Pods on Nodes.
+
+## Scheduling
+
+* [Kubernetes Scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/)
+* [Assigning Pods to Nodes](/docs/concepts/scheduling-eviction/assign-pod-node/)
+* [Pod Overhead](/docs/concepts/scheduling-eviction/pod-overhead/)
+* [Taints and Tolerations](/docs/concepts/scheduling-eviction/taint-and-toleration/)
+* [Scheduling Framework](/docs/concepts/scheduling-eviction/scheduling-framework)
+* [Scheduler Performance Tuning](/docs/concepts/scheduling-eviction/scheduler-perf-tuning/)
+* [Resource Bin Packing for Extended Resources](/docs/concepts/scheduling-eviction/resource-bin-packing/)
+
+## Pod Disruption
```

Comment:
Add a short description?

Reply:
The glossary definition acts as the short description in this case (see staging at https://deploy-preview-27739--kubernetes-io-master-staging.netlify.app/docs/concepts/scheduling-eviction/#pod-disruption)

Reply:
nit: maybe we could add an extra line saying something like:

`These are the types of disruptions the control plane can apply to pods:`

And should we reorder `Preemption` to be before `Priority`? 🤔

Reply:
I think instead we call the link Pod Preemption and link to the Preemption section of that page?

Reply:
sgtm

Reply:
no wait, but this is the index, it should have the full name.